html/php-pre-challenge2/index.php

```diff
@@ -1,12 +1,16 @@
 <?php
 $array = explode(',', $_GET['array']);
-
 // 修正はここから
 for ($i = 0; $i < count($array); $i++) {
-
+    for ($j = 0; $j < count($array)-1; $j++) {
```

Comment:
```suggestion
    for ($j = 0; $j < count($array)-$i-1; $j++) {
```
こうすると内側の無駄な(既にソート済みの部分の)ループを減らせます。

Reply:
ありがとうございます！